etc/env/base/private.toml

```diff
@@ -9,6 +9,7 @@ test_database_prover_url = "postgres://postgres:notsecurepassword@localhost:5433
 [eth_sender.sender]
 # Set in env file for development, production, staging and testnet.
 operator_private_key = "0x27593fea79697e947890ecbecce7901b0008345e5d7259710d0dd5e500d040be"
+reverter_private_key = "0xe131bc3f481277a8f73d680d9ba404cc6f959e64296e0914dded403030d4f705"
```

Comment:
comment how this differs. (and why do we even need a separate key for the reverter and cannot simply reuse the operator key)

Reply:
You were right, we can use the same key